content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -67,12 +71,16 @@ enum Type {
 
 ## Constraint and index names (Preview)
 
-In [2.29.0](https://github.com/prisma/prisma/releases/tag/2.29.0) and later, you can use the `map` field to define the **underlying constraint and index names** in the Prisma schema. The `map` field is _only_ visible in the schema if you use a constraint name that differs from Prisma's naming convention.
+In [2.29.0](https://github.com/prisma/prisma/releases/tag/2.29.0) and later, you can use the `map` argument to define the **underlying constraint and index names** in the Prisma schema. This means the `@id`, `@@id`, `@unique`, `@@unique`, `@index` and `@relation` attributes now take an optional argument called `map`. The `map` argument will _only_ be rendered in the schema if you use a name that differs from Prisma's naming convention.
+
+<!-- todo this can be formatted nicer, and not sure this is the correct TS client syntax -->
+
+Additionally, the `@@id` and `@@unique` attributes take an optional `name` argument that allows you to customize your Prisma client API. Specifying `@@id([firstName, lastName], name: "fullName")` will allow you to do `prisma.user.findUnique({where: {fullName:{firstName: "Paul", lastName:"Panther"}}})` instead of `prisma.user.findUnique({where: {firstName_lastName:{firstName: "Paul", lastName:"Panther"}}})`.
```

Comment:
cc @Jolg42 Can you take a look at the TS call here if that is correcT? Thanks.

Reply:
Looks correct but might a bit weird in one line without any space?
Maybe better to have 2 small code blocks?

Reply:
This should definitely be split into code blocks.